tests/classify_test.py

```diff
@@ -128,3 +128,20 @@ def test_application_directories(in_tmpdir, no_empty_path):
     # Should be application with extra directories
     ret = classify_import('testing', application_directories=('.', 'tests'))
     assert ret is ImportType.APPLICATION
+
+
+def test_override_absent_application_modules(in_tmpdir, no_empty_path):
+    # Should be classified 3rd party without argument
+    ret = classify_import('c_module')
+    assert ret is ImportType.THIRD_PARTY
+    # Should be classified application with the override
+    ret = classify_import(
+        'c_module', override_absent_application_modules=('c_module',),
+    )
+    assert ret is ImportType.APPLICATION
+    # Trying to force __future__ to be application should error
+    with pytest.raises(ValueError):
+        ret = classify_import(
+            'c_module',
+            override_absent_application_modules=('__future__',),
+        )
```

Comment:
a test should just test one thing, you've got two tests here

Reply:
Indeed! Split up.